marklogic-data-hub/src/main/java/com/marklogic/hub/step/impl/StepRunnerUtil.java

```diff
@@ -56,4 +57,15 @@ protected static void initializeStepRun(JobDocManager jobDocManager, RunStepResp
         }
     }
 
+    protected static String objectToString(Object obj) {
```

Comment:
Since this is a convenience function, I think it's worth checking for null here and likely returning null by default, as it should be up to the client to first throw an error if the object it's passing in is null.

Reply:
done